### PR TITLE
Enhance the VSCode "Getting Started" experience

### LINKED ISSUE #442: I want VSCode Extension user documentation
As an AaC user, I want VSCode Extension user documentation, so that I can easily determine how to use the VSCode Extension and what value it provides to me.

Blocks #443

## AC:
- [x] The documentation has new pages concerning the VSCode Extension:
  - [x] Settings overview/what they do and need
  - [x] How to setup the Extension (Getting Started)
  - [x] What features the Extension currently supports
    - [x] Running AaC commands from VSCode
    - [x] LSP features
- [x] The VSCode Readme is updated with high-level items
  - [x] Settings - what they do/need
  - [x] Features
    - [x] LSP
    - [x] Commands

### LINKED ISSUE #443: I want a walkthrough for the VSCode Extension
As an AaC user, I want a walkthrough for the VSCode Extension, so that I have an immediate, helpful guide when I start the extension.

(Soft) blocked by #442 

https://code.visualstudio.com/api/references/contribution-points#contributes.walkthroughs
https://code.visualstudio.com/api/ux-guidelines/walkthroughs
https://github.com/microsoft/vscode-extension-samples/tree/main/getting-started-sample

## AC:
- [x] The walkthrough provides the user with initial setup instructions
  - [x] Local AaC and Python installation
    - [x] Prompt the user to set the local Python path
    - [x] Prompt the user to set the local AaC path
  - [ ] AaC settings configuration
    - [ ] LSP (IO/TCP)
- [x] Include brief description of the features
  - [x] LSP Features
  - [x] AaC Commands

### PR DESCRIPTION
**Pull request by cvchaparro**
Closes #442
Closes #443

Changes:
- Add a walkthrough for getting started with the AaC VSCode extension
- Add documentation on the VSCode extension describing provided features,
  configuration settings, etc.
- Remove the `pythonPath` and `version` configuration settings from the VSCode
  extension.
- Update the extension such that it handles cycling the LSP server when the
  `aacPath` is set without having to restart VSCode.

### CODE REVIEW

File: vscode_extension/src/test/suite/aacExecutableWrapper.test.ts [hunk 1]
```diff
@@ -1,24 +1,2 @@
-import * as assert from "assert";
-
-import * as aacWrapper from "../../aacExecutableWrapper";
-import * as configuration from "../../configuration";
-
 suite("AaC Executable Wrapper Test Suite", () => {
```
Comment (cvchaparro) [inline vscode_extension/src/test/suite/aacExecutableWrapper.test.ts:1]: We'll need to add some more tests pretty soon.

Comment (Coffee2Bits) [reply vscode_extension/src/test/suite/aacExecutableWrapper.test.ts:1]: Ugh, yeah. I think the extension is slim enough where it's not that big of a deal, but I would certainly feel better if we had the selenium test runner working. :/

**Review by Coffee2Bits — CHANGES_REQUESTED** [review]
Great work! Love to see more documentation.

I left a couple suggested edits, but you're there aside from the other issue you're tracking.

**Review by Coffee2Bits — APPROVED** [review]
Good work!

**Pull request merged**